Homework5_SvetlanaNesenchuk/Task5/ArgumentsProcessing.py

```diff
@@ -0,0 +1,72 @@
+"""Arguments processing."""
+
+import argparse
+titles = []
+years = []
+ratings = []
+try:
+    count = 0
+    with open('ratings.list', 'r', encoding='ISO-8859-1') as file:
+        for line in file:
+            if 'New  Distribution  Votes  Rank  Title' in line:
+                break
+        for line in file:
+            if count < 250:
+                movie = line.strip().replace('(', ' ').split('  ')
+                titles.append(movie[3])
+                years.append(movie[4].rstrip('/I)'))
+                ratings.append(movie[2].lstrip())
+                count += 1
+except FileNotFoundError:
+    print("This file doesn't exist")
+
+year_count = {year: years.count(year) for year in years}
+rating_count = {rating: ratings.count(rating) for rating in ratings}
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--year', help='displays Top250 movies titles with year',
```

Comment:
if you define arguments in this way I should print `python main.py --year whatever` to print titles with year. You can use `action="store_true"` option instead. 
It also concerns --rate and --all keys. 
You can find examples in presentation for 5th lesson